homework/english/raby_alexandr/lection1/vocabulary.md

```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
+2. I create my first homework in Github.
+3. Cannot connect to the specified server.
+4. We are debuging their code now. 
+5. She is developing new application at the moment.
+6. He disables his PC before going to bed.
```

Comment:
In this case u may to add "every day" at the end of sentence

Reply:
Perhaps this is appropriate.